### DISCUSSION TITLE
Docs on custom Redis configs - string vs. symbol keys

### DISCUSSION THREAD

**Discussion by franzliedke** [sidekiq/sidekiq#5165] [Q&A] [answered]
The [documentation on configuring Redis in an initializer](https://github.com/mperham/sidekiq/wiki/Using-Redis#using-an-initializer) explicitly states:

> **NOTE:** The configuration hash *must* have symbolized keys.

This [does not seem to be the case anymore](https://github.com/mperham/sidekiq/blob/cd553fa14cefe789684f3cf35b0f61e4e22b6305/lib/sidekiq/redis_connection.rb#L11). Do I assume correctly that this can be removed? Didn't want to edit the wiki without asking... 😃

**Comment by mperham** [comment, marked as answer]
If they’ve fixed it, yeah you can remove that.

**Reply by franzliedke** [reply]
The `redis` gem may still have that requirement. But a workaround for this was added on Sidekiq's side (see the link in my original post), apparently in PR #2989.

**Reply by franzliedke** [reply]
As that likely won't change your answer, I made the change. 😎 Thanks!
